content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -95,42 +86,52 @@ There may be cases when a workspace contains conflicting variables of the same t
 
-### 3. Global Variable Sets
+### 5. Global Variable Sets
 
 Non-global variable sets always take precedence over global variable sets that are applied to all workspaces within an organization. This is true regardless of when the global variable set was applied.
 
-### 4. Variable Files
-
-When variables from [committed `.auto.tfvars` files](#loading-variables-from-files) have the same key as existing variables in the Terraform Cloud workspace, the variables applied to the workspace overwrite variables from the files.
-
+### 6. `auto.tfvars` Variable Files
 
+Variables in the Terraform Cloud workspace and variables provided through the command line always overwrite variables from files ending in `auto.tfvars` with the same key.
 
 ### Precedence Example
 
 Consider an example workspace that has the following variables applied:
 
-| Name | Scope | Date Applied | ACCESS_KEY | ACCESS_ID | VAR1 | KEY1 | VAR2|
-|-----|-------|-------|-----|-------|-------|--------|--------|------|
-| Variables | Workspace-Specific | 10/1 | `g47fh474` | `874hf7u4` | `h` |     |     |
-| Variable Set A | Non-Global | 10/4 |   |   | `y` | `x` |   |   |
-| Variable Set B | Global | 10/20 |   |     |   |  `z` | `a` |
+| Name           | Scope              | Date Applied | ACCESS_KEY | ACCESS_ID  | VAR1 | KEY1 | VAR2 | foo |
+|----------------|--------------------|--------------|------------|------------|------|------|------|-----|
+| TF_VAR_foo=8   | Run-Specific       |              |            |            |      |      |      | `8` |
+| -var="foo=9"   | Run-Specific       |              |            |            |      |      |      | `9` |
+| Variables      | Workspace-Specific | 10/1         | `g47fh474` | `874hf7u4` | `h`  |      |      | `1` |
+| Variable Set A | Non-Global         | 10/4         |            |            | `y`  | `x`  |      | `2` |
+| Variable Set B | Global             | 10/20        |            |            |      | `z`  | `a`  | `3` |
 
 When you trigger a run, Terraform Cloud applies the following variables:
 
+- **Run-Specific:** `foo` is specified in all scopes. The -var value takes precedence over all other values, including the run-specific TF_VAR_foo values.
+
 - **Workspace-Specific:** `ACCESS_KEY`, `ACCESS_ID`, and `VAR1`. That means `VAR1` equals `h` for this run, overwriting the value in Variable Set A.
 
 - **Variable Set A:** `KEY1`. That means `KEY1` equals `x` for this run, overwriting the value in Variable Set B.
 
 - **Variable Set B:** `VAR2`. This is a global variable set, so Variable Set A takes precedence regardless of when it was applied.
 
-![An example scenario demonstrating variable precedence in Terraform Cloud](/docs/cloud/workspaces/images/variable-precedence-example.png)
\ No newline at end of file
+![An example scenario demonstrating variable precedence in Terraform Cloud](/docs/cloud/workspaces/images/variable-precedence-example.png)
```

Comment:
I am going to send you a new image we can use on slack! Please replace the current one with whatever the final visual is (once we settle on the new variable names, etc.)